tests/test_attributes.py

```diff
@@ -57,36 +54,17 @@ def test_attributes(browser, netid, utils, sp_url, sp_domain, secrets, test_env)
     }
 
     with utils.using_test_sp(sp):
-        fresh_browser.set_window_size(1024, 768)
+        browser.set_window_size(1024, 768)
         # go to url to check saml properties
         # https://diafine6.sandbox.iam.s.uw.edu/shib{test_env}/server-vars.aspx
-        fresh_browser.get(f'{sp_url(sp)}/shib{test_env}')
-        fresh_browser.send_inputs(login, password)
-        fresh_browser.click(Locators.submit_button)
-        fresh_browser.wait_for_tag('h2', f'{sp_domain(sp)} sign-in success!')
-        fresh_browser.get(f'{sp_url(sp)}/shib{test_env}/server-vars.aspx')
-        fresh_browser.wait_for_tag('pre', 'cn')
-        content = fresh_browser.find_element_by_tag_name('pre')
+        url = sp_shib_url(sp)
+        browser.get(url)
+        log_in_netid(browser, login)
+        browser.get(f'{url}/server-vars.aspx')
+        content = browser.wait_for_tag('pre', 'cn')
         content_clean = content.get_attribute('innerHTML')
         attribute_data = [item.strip() for item in content_clean.split("<br>")]
 
-        # eduPersonScopedAffiliation is a unique case
-        wanted_key = 'eduPersonScopedAffiliation'
-
-        result = next(filter(lambda x: x.startswith(wanted_key), attribute_data))
-        found_values = set(result.split("= ")[1].split(";"))
-
         for key, value in attributes_to_test.items():
-            if key == wanted_key:
-                assert found_values == {
-                    'employee@washington.edu',
-                    'student@washington.edu',
-                    'staff@washington.edu',
-                    'member@washington.edu',
-                }, f'Unexpected attributes found in comparison with: {found_values}'
-            else:
-                # matches on exact string only. mail won't count as a match for uwEduEmail and uwEduEmail won't match
-                # mail
-                assert f'{key} = {value}' in attribute_data
-
-        fresh_browser.close()
```

Comment:
Lines 73-78 and 80-86 should not be removed, they account for testing `eduPersonScopedAffiliation`.

Reply:
I feel like this might have been a merge conflict problem. I'll restore the intent here. I wouldn't have purposely deleted that stuff.

Reply:
I think these weren't so much "removed" as "accidentally merge-conflicted away" --- i'll make sure to add those test cases back

Reply:
Thx